config/routes.rb

```diff
@@ -1,56 +1,6 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  post '/rates', to: 'shipping_api#rates'
+  post '/logs/new', to: 'shipping_api#create_log'
```

Comment:
In looking at the controller it seems like you are creating a log object and saving it. So in the rails restful design, that would usually just be `post '/logs'`.